rpki-validator/src/main/resources/application.properties

```diff
@@ -92,3 +92,8 @@ management.endpoints.enabled-by-default=false
 management.endpoint.prometheus.enabled=true
 management.endpoints.web.exposure.include=prometheus
 
+# Enabling strict validation will not allow stale manifest, stale crl, and reject manifest with entries referring to non-existent objects.
+rpki.validator.strict-validation=false
+
+# This disables RRDP, so the validator only uses rsync to download repositories.
+rpki.validator.rsync-only=false
```

Comment:
Just checking: there is no corresponding rrdp only?

Reply:
Nope.

Reply:
> Just checking: there is no corresponding rrdp only?

We probably don't need "rrdp only" because choosing rrdp is the default behaviour.
And we do it for the testbed, so it should be enough to have a "default" and "rsync only" versions.

Reply:
It does not feel super useful, but it would make it possible to check what goes missing without fallback.

Routinator covers this way of checking.